pkg/agent/containerd/runwasi_test.go

```diff
@@ -0,0 +1,195 @@
+//go:build linux
+// +build linux
+
+package containerd
+
+import (
+	"io/fs"
+	"reflect"
+	"testing"
+	"testing/fstest"
+)
+
+func Test_UnitFindWasiContainerRuntimes(t *testing.T) {
+	executable := &fstest.MapFile{Mode: 0755}
+	type args struct {
+		root fs.FS
+	}
+	tests := []struct {
+		name string
+		args args
+		want runtimeConfigs
+	}{
+		{
+			name: "No runtimes",
+			args: args{
+				root: fstest.MapFS{},
+			},
+			want: runtimeConfigs{},
+		},
```

Comment:
```suggestion
	tests := []struct {
		name string
		arg fs.FS
		want runtimeConfigs
	}{
		{
			name: "No runtimes",
			arg: fstest.MapFS{},
			want: runtimeConfigs{},
		},
```

Reply:
When only a single argument exists, no need to embed it into a struct.

Reply:
I've now added another arg, see e4d26eb086f869e3486ec33ae74daecd9faf1def